docs/sources/trackers/google-tag-manager/snowplow-template/index.md

```diff
@@ -0,0 +1,6 @@
+---
+title: Snowplow Template
+sidebar_position: 200
+---
+
+This template implements the [Snowplow JavaScript tracker](/docs/sources/trackers/javascript-trackers/web-tracker/index.md) for Google Tag Manager. It allows for the sending of [Snowplow events](docs/sources/trackers/snowplow-tracker-protocol/index.md) from your website to your Snowplow collector.
```

Comment:
this page doesn't have much on it, can you combine it with the Events subsection please